app/src/lib/app-state.ts

```diff
@@ -210,7 +210,12 @@ export enum PopupType {
 
 export type Popup =
   | { type: PopupType.RenameBranch; repository: Repository; branch: Branch }
-  | { type: PopupType.DeleteBranch; repository: Repository; branch: Branch }
+  | {
+      type: PopupType.DeleteBranch
+      repository: Repository
+      branch: Branch
+      aheadBehind: IAheadBehind | null
```

Comment:
Instead of passing `IAheadBehind` let's pass a Boolean value, `existsOnRemote` that determines whether or not we show the delete on remote dialog.

Reply:
Fixed in 60f791b8705928d35ba0bb8400edc0e404bcbc5a. 😺